dlg/README

```diff
@@ -0,0 +1,26 @@
+# NPC Dialog Management
+
+These files manage random and triggered NPC dialog, primarily managed through `.tab` files and converted into admin commands using `dialogconvert.py`.
+
+## Setting Up Dialog
+
+After recreating the world (e.g., `send o 0 recreate all`) or during initial startup:
+
+1. Move `dialog.txt` to the server run directory
+1. Execute the admin command `read dialog.txt` to load NPC dialog into the speech library.
```

Comment:
All numbered 1?

Reply:
I did this with GitHub Markdown rendering in mind, but this isn't a Markdown file! I'll update.

Reply:
Fixed.